### DISCUSSION TITLE
How can I add flex direction?

### DISCUSSION THREAD

**Discussion by iroshandezilva** [webstudio-is/webstudio-community#43] [Q&A]
I'm trying to add `flex-direction: column-reverse;` to a responsive layout, but I can't seem to find it in the layout section. Do you have any ideas on how I can achieve this?

**Comment by markxtang** [comment]
We're working on the row/column-reverse implementation. In the meanwhile, you can select the child element, and use the Flex Child>Order styling to achieve it
![image](https://github.com/webstudio-is/webstudio/assets/122043657/be7a3008-83cc-45c5-bc65-afe2d4605cfd)
